src/cypress/utils/navigation.ts

```diff
@@ -11,9 +11,9 @@ interface NavigationOptions {
   pageIndex: number
 }
 
-export function goToSearchPage(options?: NavigationOptions): boolean {
+export function goToSearchPage(options: NavigationOptions) {
```

Comment:
This function seems app-specific, maybe it could be a custom command inside the app that uses the lib (or `util` inside the examples folder, maybe it should not be exported, but I'm not sure if someone is using it). what do you think?